tests/DBAL/AdaptorTest.php

```diff
@@ -23,16 +25,17 @@ class AdaptorTest extends \PHPUnit_Framework_TestCase {
 	private $adaptor;
 
 	/**
-	 * @var \PHPUnit_Framework_MockObject_MockObject
+	 * @var MockObject
 	 */
 	private $connection;
 
 	/**
-	 * @var \PHPUnit_Framework_MockObject_MockObject
+	 * @var MockObject
```

Comment:
```suggestion
	 * @var MockObject&Statement
```

En oubliant pas le `use \Doctrine\DBAL\Driver\Statement`